test/clearingHouse/ClearingHouse.closePosition.test.ts

```diff
@@ -765,8 +765,10 @@ describe("ClearingHouse closePosition", () => {
             // set MaxTickCrossedWithinBlock so that trigger over price limit
             await exchange.setMaxTickCrossedWithinBlock(baseToken.address, 250)
 
-            // after alice partial close, market price: 165.664, index price: 165, spread: 0.44% (nor over price band limit)
-            await expect(closePosition(fixture, alice)).to.be.revertedWith("EX_OPLAS")
+            // after alice partial close, market price: 165.893, index price: 165
+            // spread: 0.54% (nor over price band limit)
+            // positionSize remaining: 0.3815690374
```

Comment:
convert this comment to an `expect`?

Reply:
fixed in https://github.com/perpetual-protocol/perp-curie-contract/pull/90/commits/fb025ed686e167212354f8af67645b56edf9222d